### ISSUE TITLE
"put-on-slit" failed

### ISSUE THREAD

**Issue by astronomerdave** [CaltechOpticalObservatories/NGPS#174]
```
==> /data/logs/slicecamd_20241031.log <==
2024-10-31T16:42:25.458  (Common::DaemonClient::send) TIMEOUT polling socket localhost/9009 on fd 18 for acamd
2024-10-31T16:42:25.458  (Slicecam::Interface::get_acam_guide_state) ERROR getting guiding state from ACAM
2024-10-31T16:42:25.458  (Slicecam::Interface::put_on_slit) ERROR getting guide state

```

**Comment by astronomerdave** [comment]
blocking port 9009 may not have been released?

**Comment by astronomerdave** [comment]
sequencer is using the blocking port!

**Comment by astronomerdave** [comment]
resolved with commit 36685bc19e77b68a91aa45d0df44b82af22d6644
